handlers.go

```diff
@@ -3366,9 +3374,8 @@ type getApiKeysResp struct {
 // @Param        key path string true "Key"
```

Comment:
Let’s add a comment here that this endpoint will completely delete the key and will not be recoverable. We can add that to the description part of the swagger doc.